coordinator/history/history.go

```diff
@@ -118,6 +121,46 @@ func (h *History) SetLatest(oldT, newT *LatestTransition, signingKey *ecdsa.Priv
 	return nil
 }
 
+// WatchLatestTransitions starts a goroutine that sends LatestTransition structs to the returned
+// channel whenever the latest transition changes in the underlying store.
+//
+// The goroutine continues to run until either the context expires or the underlying store watcher
+// stops. In both cases, the channel will be closed.
+func (h *History) WatchLatestTransitions(ctx context.Context) (<-chan LatestTransition, error) {
+	ch, cancelWatch, err := h.store.Watch("transitions/latest")
+	if err != nil {
+		return nil, fmt.Errorf("watching latest transitions: %w", err)
+	}
+
+	transitionCh := make(chan LatestTransition)
+	go func() {
+		defer close(transitionCh)
+		defer cancelWatch()
+		for {
+			select {
+			case buf, ok := <-ch:
+				if !ok {
+					h.log.Warn("store watcher closed unexpectedly")
+					return
+				}
+				var t LatestTransition
+				if err := t.unmarshalBinary(buf); err != nil {
+					h.log.Error("store watcher sent something that's not a LatestTransition", "error", err)
+					continue
+				}
+				select {
+				case transitionCh <- t:
```

Comment:
Is this a fallthrough in this case? If so, why not use the comma syntax? Or does it not support the redirection here?

Reply:
There's neither fallthrough nor expression list in select statements:

* https://go.dev/ref/spec#Select_statements
* https://go.dev/ref/spec#Expression_switches